mappings/net/minecraft/client/font/Font.mapping

```diff
@@ -1,2 +1,4 @@
 CLASS net/minecraft/class_390 net/minecraft/client/font/Font
 	METHOD method_2040 getGlyph (I)Lnet/minecraft/class_383;
+		ARG 1 codePoint
+	METHOD method_27442 getProvidedGlyphs ()Lit/unimi/dsi/fastutil/ints/IntSet;
```

Comment:
What does this return? A set of the code points for which this font has glyphs?
Or a set of some integer related directly to the glyphs?

Reply:
Ah forget about this comment. It's a nitpick and too much bikeshedding. getProvidedGlyphs should be fine

Reply:
It's a fair point, because in the real world, glyphs and codepoints don't form a 1to1 relation. However in Minecraft they unfortunately do.